modules/logging-upgrading-clo.adoc

```diff
@@ -16,6 +16,39 @@ To update the {clo} to a new major release version, you must modify the update c
 
 .Procedure
 
+. Optional: When upgrading to the {clo} 5.9 or later, you must add the `v13` schema version to the `LokiStack` custom resource before the upgrade:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+# ...
+spec:
+# ...
+  storage:
+    schemas: # <1>
+    - effectiveDate: "<yyyy>-<mm>-<past_dd>" # <2>
+      version: v12 # <3>
+    - effectiveDate: "<yyyy>-<mm>-<future_dd>" # <4>
+      version: v13 # <5>
+# ...
+----
+<1> Upgrading to the {clo} 5.9 or later without the `v13` schema version in the `LokiStack` custom resource will result in a web console warning.
```

Comment:
I feel the example and comments should be maintained the simplest possible. 

Then, the sentence below should go better out of the explanation of the fields in the example:

_"Upgrading to the {clo} 5.9 or later without the `v13` schema version in the `LokiStack` custom resource will result in a web console warning."_ 

It could be inside or to a continuation of the sentence:

 _"Optional: When upgrading to the {clo} 5.9 or later, you must add the v13 schema version to the LokiStack custom resource before the upgrade"_

But, if we try to join both phrases to be only one, it could be:

_"Optional: When upgrading to the {clo} 5.9 or later, you must add the v13 schema version to the LokiStack custom resource before the upgrade or it will result in a web console warning":_

And in this way, you could remove the explanation from the example

Reply:
I've moved to after the yaml block.
I wouldn't add it into the main sentence of the step to avoid making it confusing for readers.

Reply:
> I feel the example and explanations should be maintained the simplest possible.

+1

Thank you for your feedback. I've rewritten these lines.